content/webapp/services/prismic/transformers/body.ts

```diff
@@ -439,21 +424,25 @@ export function transformContentListSlice(
         .map(content => {
           switch (content.type) {
             case 'pages':
-              return transformPage(content);
+              return transformPage(content as unknown as PagesDocument);
```

Comment:
This one too I'd love to understand better, why is this needed?

Reply:
The content can be one of many types. Each of the transform functions expects a particular one of those. Because of the switch statement we know what type we are dealing with so I tell typescript that is so it doesn't complain. Because the different possible types don't overlap sufficiently typescript complains unless it is converted to unknown first.

Another way to do it would be to write some type guards to identify each specific specific type.